app/views/planning_application/assessment_details/_form.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= render "planning_application/assessment_details/#{category}/information", category: category %>
+
+<%= form_with model: [@planning_application, @assessment_detail], local: true, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+  <%= form.govuk_error_summary(presenter: ErrorPresenter) %>
+
+  <%= form.govuk_text_area :entry, label: nil, rows: 10 %>
+
+  <%= form.hidden_field :category, value: category %>
+
+  <div class="govuk-button-group">
+    <%= form.submit "Save and mark as complete", class: "govuk-button", data: { module: "govuk-button" } %>
+    <%= form.submit "Save and come back later", class: "govuk-button govuk-button--secondary", data: { module: "govuk-button" } %>
+
+    <%= link_to "Back", :back, class: "govuk-button govuk-button--secondary" %>
```

Comment:
Now you can use the [`back_link`](https://github.com/unboxed/bops/blob/main/app/helpers/application_helper.rb#L6) helper 😁